ptz: true
# Internal configuration of the camera. It can instantly flip the camera head 180 degrees and continue to pan beyond its zero point
autoflip: false
# Internal configuration of the camera. It can automatically flip the image electronically 180 degrees, and continue to follow the person in the correct orientation, without delay.
eflip: false
#
#pan_joint: a15_p5564_ptz_camera_pan_joint
#tilt_joint: a15_p5564_ptz_camera_tilt_joint
# degrees
min_pan_value: -170
max_pan_value: 170
min_tilt_value: 0
max_tilt_value: 90
home_pan_value: 0
home_tilt_value: 90

# Flag to use a timeout to allow other processes to control the camera (like the Axis Web)
use_control_timeout: false
control_timeout_value: 5.0

